protocol/0056-REWA-rewards_overview.md

```diff
@@ -51,7 +53,82 @@ These metrics apply only to the sum of fees for the epoch in question.
 That is, the metrics are reset to zero for all parties at the end of the epoch.
 If the reward account balance is 0 at the end of the epoch for a given market, any parties with non-zero metrics will not be rewarded for that epoch and their metric scores do not roll over (they are still zeroed).
 
-Market activity (fee based) reward metrics (the total fees paid/received by each party as defined above) are stored in [LNL checkpoints](./0073-LIMN-limited_network_life.md) and are restored after a checkpoint restart to ensure rewards are not lost.
+Fee-based reward metrics (the total fees paid/received by each party as defined above) are stored in [LNL checkpoints](./0073-LIMN-limited_network_life.md) and are restored after a checkpoint restart to ensure rewards are not lost.
+
+### Average position metric
+
+The average position metric, $m_{ap}$, measures each parties time-weighted average position over a number of epochs.
+
+At the start of each epoch, the network must reset each parties time weighted average position for the epoch ($\bar{P}$) to `0`. Whenever a parties position changes during an epoch, **and** at the end of the epoch, this value should be updated as follows.
+
+Let:
+
+- $\bar{P}$ be the parties time weighted average position in the epoch so far
+- $P_{n}$ be the parties position before their position changed
+- $t_{n}$ be the time the party held the previous position in seconds
+- $t$ be the amount of time elapsed in the current epoch so far
+
+
+$$\bar{P} = \bar{P} \cdot \left(1 - \frac{t_{n}}{t}\right) + \frac{|P_{n}| \cdot t_{n}}{t}$$
+
+At the end of the epoch, the network must store the parties time weighted average position and then calculate their average position reward metric as follows.
+
+Let:
+
+- $m_{ap}$ be the parties average position reward metric
+- $\bar{P_{i}}$ be the parties time weighted average position in the i th epoch
+- $N$ be the window length specified in the recurring transfer.
+
+$$m_{ap} = \frac{\sum_{i}^{n}\bar{P_{i}}}{N}$$
+
+### Relative return metric
+
+The relative return metric, $m_{rr}$, measures each parties average relative return, weighted by their [time-weighted average position](#average-position-metric), over a number of epochs.
+
+At the end of each epoch, the network must calculate and store the parties relative returns as follows.
+
+Let:
+
+- $r_i$ be the parties relative returns in the epoch
+- $m2m_{wins}$ be the sum of all mark-to-market win transfers in the epoch
+- $m2m_{losses}$ be the sum of all mark-to-market loss transfers in the epoch
+- $TWAP$ be the parties time-weighted average position in the epoch.
+
+$$r = \frac{m2m_{wins} + m2m_{losses}}{TWAP}$$
```

Comment:
```suggestion
$$r = \frac{m2m_{wins} - m2m_{losses}}{TWAP}$$
```

Reply:
In fact we should probable clarify whether we expect $m2m_{\text{losses}}$ to come with a negative sign above...

Reply:
Good point, I think since it will be the sum of the "loss" transfers it should be a positive value. For extra clarity, though I will add abs signs to both.
```suggestion
$$r = \frac{|m2m_{wins}| - |m2m_{losses}|}{\bar{P}}$$
```